class-wc-calypso-bridge-shared.php

```diff
@@ -145,6 +152,13 @@ public function add_ecommerce_plan_styles() {
 			wp_enqueue_style( 'wp-calypso-bridge-ecommerce-navigation', WC_Calypso_Bridge_Instance()->get_asset_path() . 'assets/css/ecommerce-navigation.css', array(), WC_CALYPSO_BRIDGE_CURRENT_VERSION );
 		}
 	}
+
+	/**
+	 * Add styles for ecommerce plan trial.
+	 */
+	public function add_ecommerce_trial_plan_styles() {
+		wp_enqueue_style( 'wp-calypso-bridge-ecommerce-trial', WC_Calypso_Bridge_Instance()->get_asset_path() . 'assets/css/free-trial-admin.css', array(), WC_CALYPSO_BRIDGE_CURRENT_VERSION );
```

Comment:
Does this add style to all pages? If so, what do you think we only add it to payment pages?

Reply:
@chihsuan Good question. It gets included when customer is in trial mode. I was thinking about using this CSS for all non-React trial related changes where we can't use scss.

Reply:
Ah, got it!